etc/bin/run.sh

```diff
@@ -37,5 +37,10 @@ then
    SPARK_OPTS=${__TOREE_SPARK_OPTS__}
 fi
 
+if [ "${TOREE_OPTS}" = "" ]
+then
+   TOREE_OPTS=${__TOREE_OPTS__}
+fi
+
 SPARK_OPTS="--driver-class-path=\"${TOREE_ASSEMBLY}\" ${SPARK_OPTS}"
-eval exec "${SPARK_HOME}/bin/spark-submit" "${SPARK_OPTS}" --class org.apache.toree.Main "${TOREE_ASSEMBLY}" "$@"
+eval exec "${SPARK_HOME}/bin/spark-submit" "${SPARK_OPTS}" --class org.apache.toree.Main "${TOREE_ASSEMBLY}" "${TOREE_OPTS}" "$@"
```

Comment:
`TOREE_OPTS` should be env setting and `$@`. The next question is in what order?